components/brave_federated/config_utils.cc

```diff
@@ -0,0 +1,133 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_federated/config_utils.h"
+
+#include "brave/components/brave_federated/features.h"
+#include "brave/components/brave_federated/task/model.h"
+
+#include "base/json/json_reader.h"
```

Comment:
why are these includes separated from the brave includes? Please alphabetize except for the config_utils.h https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes